### ISSUE TITLE
Use unknown instead of any

### ISSUE THREAD

**Issue by andrenarchy** [drwpow/openapi-typescript#554]
It would be great if the resulting ts would not use `any` but `unknown` instead.

**Comment by drwpow** [comment]
Thanks for the suggestion. Do you have an example of a schema you’re using, what it’s generating, and what you’d like it to be instead?

Also would you be open to opening a PR for this?

**Comment by andrenarchy** [comment]
One example is a schema I have with the following property somewhere:
```
          "presets": {
            "description": "JSON structure containing the preset value for created/updated items.",
            "type": "object",
            "nullable": true
          },
```
This gets transformed into:
```
presets?: { [key: string]: any } | null
```

The use of `any` is generally discouraged since it can be accessed in any way. But that's not really what we want for a type where we don't have further knowledge. So the better option would be to use `unknown` for all types that are currently generated as `any` because then it's up to the user to do further checks on the type. In the above example I'd expect:
```
presets?: { [key: string]: unknown } | null
```

See also [eslint](https://github.com/typescript-eslint/typescript-eslint/blob/v4.24.0/packages/eslint-plugin/docs/rules/no-explicit-any.md) (which is complaining about `any` in the `recommended` rule set) and the [TypeScript announcement](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#new-unknown-top-type) that says 

> `unknown` is the type-safe counterpart of `any`

which really is the essence I think. :)

**Comment by drwpow** [comment]
Thanks for providing more info. I also see that #625 was opened to address this. If it’s all the same to people, I don’t see why `unknown` isn’t an improvement. I’m on board with this change.

**Comment by drwpow** [comment]
Added in #769!
